ResultPicker.js

```diff
@@ -1,49 +1,62 @@
-// modified version of https://superuser.com/a/1235058/69589
-
-// find all the search results
-const querySelector = 'div.yuRUbf';
+// from https://github.com/RobinH-J/SearchingShortcuts
 
+// selector to find all Google search results. Simplified Google html:
+//   <div class="yuRUbf">
+//     <a href="The real url is here">
+//       <br>
+//       <h3>The big text we see in Google Search results</h3>
+//     </a>
+//   </div>
+// And sometimes we get pseudo-results (like Ads) that don't contain the h3: exclude them.
+const querySelector = 'div.yuRUbf h3';
 document.selectedResultId = 0;
+
 function selectResult(newId) {
-    els = document.querySelectorAll(querySelector)
-    if (newId < 0 || newId >= els.length) {
-        return  //Could modify for page nav...?
-    }
-    rp = document.getElementById("result-pointer");
```

Comment:
got rid of the result-pointer in favor of just using focus()

The result-pointer was nicer, but the only way i got it working was on top of other text. It's possible more thought and experimentation could make it work, but just relying on focus() worked, makes for simple code, and hopefully insulates us from future issues like this.